test-data/ReactA11yProptypes/FailingTestInputs/notAllowUndefined.tsx

```diff
@@ -0,0 +1,4 @@
+import React = require('react');
```

Comment:
The expression { } is undefined for our function isUndefined(), but there have an error "error TS17000: JSX attributes must only be assigned a non-empty 'expression'".
Is is tslint error or other error ? I don't know. @ipip2005 @HamletDRC